network/nxos/nxos_command.py

```diff
@@ -46,6 +52,19 @@
         by the configured retries, the task fails.  See examples.
     required: false
     default: null
+    aliases: ['waitfor']
+    version_added: "2.2"
+  match:
+    description:
+      - The I(match) argument is used in conjunction with the
+        I(wait_for) argument to specify the match policy.  Valid
+        values are C(all) or C(any).  If the value is set to C(all)
+        then all conditionals in the wait_for must be satisfied.  If
+        the value is set to C(any) then only one of the values must be
+        satisfied.
+    required: false
+    default: all
+    version_added: "2.2"
   retries:
     description:
       - Specifies the number of retries a command should by tried
```

Comment:
`I(wait_for)`

We should standardise on the new name.

Looks two lines below this, GutHub only lets you comment around the diff, not on any line.